modin/pandas/test/dataframe/test_iter.py

```diff
@@ -257,6 +257,17 @@ def test___setattr__():
 
     df_equals(modin_df, pandas_df)
 
+    # Use case from issue #4577
+    pandas_df = pandas.DataFrame([[1]], columns=["col0"])
+    modin_df = pd.DataFrame([[1]], columns=["col0"])
+
+    pandas_df.col0 = [3]
+    modin_df.col0 = [3]
+
+    pandas_df.col0.ffill()
+    modin_df.col0.ffill()
+    df_equals(modin_df, pandas_df)
+
```

Comment:
I would suggest turning it into a separate test

Reply:
+1, a new test would be good

Reply:
Sounds good, I kept it as part of this because I felt like it is testing similar code paths, but I can do that as well.